packages/FormElement/src/FormElement.js

```diff
@@ -36,13 +36,10 @@ const propTypes = {
   /** Should label be hidden, default is false. Note: this is discouraged because of accessibility requirements. */
   isLabelVisuallyHidden: PropTypes.bool,
 
-  /** Should be read-only or not, default is false. */
-  isReadOnly: PropTypes.bool,
-
   /** Label text of this field. */
   label: PropTypes.string.isRequired,
```

Comment:
I know I said we should wait for another PR, and maybe we still should, but if this `label` prop was optional and we had another prop `legend` that would add a `<fieldset>` wrapper and put the `legend` in a `<legend>` then I think we'd be all set up for checkboxes and radios.

Reply:
Or maybe we should keep this one as `isRequired` and add a `hasMultipleInputs` prop that does the same thing?

Reply:
Yeah if we do not include the `label` text then there would be no title at the top of the component, because that is what it is used for.. I think we should always show that in this component